en-US/I.xml

```diff
@@ -134,24 +134,41 @@
 			<term role="true">Intel 64</term>
 			 <listitem>
 				<para>
-					Correct.
+					<emphasis>n.</emphasis> The Intel 64-bit version of the x86 architecture. 
+					Use this format when referring to information that is exclusive to Intel processors. 
+					For Red&nbsp;Hat products, it applies only to Red&nbsp;Hat Enterprise Linux content. 
```

Comment:
Considering clarifying "it' -- the format, or Intel processors, or something else?